store/storetest/link_metadata_store.go

```diff
@@ -108,10 +108,18 @@ func testLinkMetadataStoreSave(t *testing.T, ss store.Store) {
 
 		result := <-ss.LinkMetadata().Save(metadata)
 		require.Nil(t, result.Err)
+		assert.Equal(t, result.Data.(*model.LinkMetadata).Data, &model.PostImage{})
```

Comment:
The parameters here (and below) should be `assert.Equal(t, expected, actual)`, but they are backwards. This can be a little confusing when the test fails and reverses the expected and actual labels.

Reply:
Good catch. While I understand their ordering here in some ways, it's different enough from everything else that it ends up being a huge pain.